src/components/DownloadImageDialog/VariantSelector.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+	Box,
+	FormControl,
+	FormControlLabel,
+	FormLabel,
+	Radio,
+	RadioGroup,
+	Tooltip,
+	Typography,
+} from '@mui/material';
+import { MUILinkWithTracking } from '../MUILinkWithTracking';
+import { getOsVariantDisplayText } from './utils';
+import { VersionSelectionOptions } from './version';
+import { Lightbulb } from '@mui/icons-material';
+
+const BuildVariants = ['dev', 'prod'] as const;
+export type BuildVariant = (typeof BuildVariants)[number];
+interface VaraintSelectorProps {
+	version: VersionSelectionOptions | undefined;
+	onVariantChange: (isDev: boolean) => void;
+	variant: BuildVariant;
+}
+
+export const VariantSelector: React.FC<VaraintSelectorProps> = ({
+	version,
+	variant,
+	onVariantChange,
+}) => {
+	return (
+		<FormControl>
+			<FormLabel>Select edition</FormLabel>
+			<RadioGroup
+				aria-labelledby="variant-radio-buttons-group"
+				name="developmentMode"
+				key="varian"
+				value={variant === 'dev'}
+				onChange={(event) => {
+					onVariantChange(event.target.value === 'true');
+				}}
+			>
+				{BuildVariants.map((buildVariant, index) => {
+					const label = (
+						<Typography>{getOsVariantDisplayText(buildVariant)}</Typography>
+					);
+					const isDev = buildVariant === 'dev';
+					const isDisabled =
+						version == null ||
+						(version.hasPrebuiltVariants && !version.rawVersions[buildVariant]);
+					return (
+						<Tooltip
+							title={
+								isDisabled
+									? 'This edition is not available for the selected version'
+									: undefined
+							}
+							key={index}
+						>
+							<Box display="flex" flexDirection="column">
+								<FormControlLabel
+									sx={{ opacity: isDisabled ? 0.6 : 1 }}
```

Comment:
Is 0.6 the opacity we use everywhere else? Asking because I believe I've seen us using 0.4 in other places (but it could easily have been for a different case)

Reply:
we should make a generic "disabled" rule in the theme adding it in design-tokens first. For now I've switched to 0.4 which is probably better.

Reply:
Agree